modules/core/src/main/java/org/apache/ignite/internal/pagemem/wal/IgniteWriteAheadLogManager.java

```diff
@@ -90,8 +90,10 @@ public WALPointer log(WALRecord entry, RolloverType rolloverType)
      * @throws IgniteCheckedException If failed to write.
      * @throws StorageException If IO exception occurred during the write. If an exception is thrown from this
      *      method, the WAL will be invalidated and the node will be stopped.
+     * @return Last WAL position which was flushed to WAL segment file. May be greater than or equal to a {@code ptr}.
```

Comment:
@return Last WAL position which was flushed to WAL segment file. Greater than or equal to a {@code ptr} if flush was successful, {@code null} otherwise.